jaxrs-clients/versions.lock

```diff
@@ -211,13 +211,13 @@
             ]
         },
         "com.squareup.okhttp3:logging-interceptor": {
```

Comment:
Let's discuss SAN vs CN verification here: the only thing that I can think would break were internal ETE tests, but we verified with a standard test certificate and they are already setup with SAN pointing to localhost and internal test domains.

As far as my understanding goes, we should already be generating correct certs in production, so I think we can close this issue?

Reply:
Yes